src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -183,6 +183,23 @@ public void localizationParameterMustIncludeAString() throws IOException {
         }
     }
 
+    @Test
+    public void localisationTestForInvalidStrings() {
+        for (String bundle : Arrays.asList("JabRef", "Menu")) {
+            for (String lang : Languages.LANGUAGES.values()) {
+                String propertyFilePath = String.format("/l10n/%s_%s.properties", bundle, lang);
+
+                // read in
+                Properties textKeys = LocalizationParser
+                        .getProperties(propertyFilePath);
+                //parse object "textKeys" to find any spaces
+                for (Map.Entry<Object, Object> entry : textKeys.entrySet()) {
+                    assertTrue("Found an invalid character in the " + lang + " localization of " + bundle + " : The " + entry.getKey().toString() + " : " + entry.getValue().toString() + " contains a space!", entry.getValue().toString().contains(" "));
```

Comment:
The statement must be inverted. We want the test to fail if there is a '` `' in the String.
Also, you only verify the values, but not the keys of the entry.
Just add another line where you assert that there are also no spaces in the key.

Reply:
@lynyus, I commit and push again. 
I correct the message with "At key", which seems to me more clear to find the line.
For the test : I use Junit. It find some failures

java.lang.AssertionError: Found an invalid character in the in localization of JabRef : proces_berpasangan At key : pairs_processed contains a space!

If I correct the P into Pairs_processed, it runs.

But I've got the same trouble in New_article , but I don't find why...

Reply:
Have you tried using assertFalse instead if assertTrue?

Reply:
Yes. It runs !